### ISSUE TITLE
[DATA] Plot discharge

### ISSUE THREAD

**Issue by charlottegiseleweil** [EPFL-ENAC/EML-Water-Portal#35]
On the same plot as "Water level", add another variable Discharge (with right additional axis), possibly separately for the river and for the pipes. 

@paolo will please provide us with the formula to calculate discharge in each case.

**Comment by AnderssonDan-epfl** [comment]
@charlottegiseleweil Yes we still want to have a graph with the discharge (calculated from water level) as in the scenarios it is the discharge that is calculated

**Comment by charlottegiseleweil** [comment]
Ok great thanks - do we have the formula to calculate discharge from water level for the measured data ?

**Comment by AnderssonDan-epfl** [comment]
@charlottegiseleweil For the discharge in the pipe you should already have the formula, if I remember right Yacine had given it to you. For the river, we are still working on it, it should be done in the coming weeks

**Comment by charlottegiseleweil** [comment]
-> Total flow graph is enough as per discussion today
